src/pages/HomePage/HomePage.jsx

```diff
@@ -1,12 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
 import HomePageHeader from '../../components/HomePageHeader/HomePageHeader';
 import Header from '../../components/Header/Header';
 
 const ListRestorans = () => {
+  const [activeButton, setActiveButton] = useState('');
+
+  function handleClick(id) {
+    setActiveButton(id);
+  }
   return (
-    <div className='home-page-header'>
+    <div className='home-page-wrapper'>
       <Header />
       <HomePageHeader />
+      <div className='div-main-homepage'>
```

Comment:
This header with days of week can be a separate component as well. Also please fix color of buttons. Red and grey are not implemented